misk-hibernate/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -432,10 +432,7 @@ internal class RealTransacter private constructor(
       check(!readOnly) { "Saving isn't permitted in a read only session." }
       return when (entity) {
         is DbChild<*, *> -> (hibernateSession.save(entity) as Gid<*, *>).id
-        is DbRoot<*> -> hibernateSession.save(entity)
-        is DbUnsharded<*> -> hibernateSession.save(entity)
-        else -> throw IllegalArgumentException(
-            "You need to sub-class one of [DbChild, DbRoot, DbUnsharded]")
```

Comment:
Is there any special logic in these 3 classes (or specifically in @DbUnsharded) that we lose by not using one of them?

Reply:
They're mostly just marker interfaces.